test_runner/regress/test_tenant_conf.py

```diff
@@ -62,6 +62,7 @@ def test_tenant_config(neon_env_builder: NeonEnvBuilder):
             log.info(f"show {env.initial_tenant}")
             pscur.execute(f"show {env.initial_tenant}")
             res = pscur.fetchone()
+            assert res is not None
```

Comment:
This doesn't look useful as we access res immediately after.

Reply:
If `res is None`, then `res.items()` will raise something like `AttributeError: 'NoneType' object has no attribute 'items'`.
Also `mypy` complains about it, I see no harm to have it fixed

Reply:
yes, not much harm, but a bit boilerplately